### PR TITLE
Project modal

### PR DESCRIPTION
**Pull request by teckim**
# In this pull request:

- Added eslint.
- Added JavaScript folder.
- Project cards are now generated automatically.
- Show project modal on `See Project` button click.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -95,6 +96,56 @@ <h2 class="headline__subtitle">LET’S CONNECT</h2>
         </ul>
       </div>
     </section>
+    <section>
+      <div class="project-modal" id="modal">
+        <div class="project-modal__container">
+          <button class="project-modal__close-button" type="button">
+            <img src="./assets/icons/close-black.svg" alt="Close modal icon" />
+          </button>
```
Comment (Mre55) [inline src/index.html:104]: When clicking the `See project` button to check project details, the popup with details about the project appears as expected, but when clicking the close (X) button, the popup is not disappearing. Kindly, fix that. Here is the screen record for that:

https://user-images.githubusercontent.com/34578631/167814999-9f570401-5af3-4697-bcc8-98e4ad79bde2.mov

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @teckim,** 

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/projects.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  // portfolioSection.innerHTML += gentateProjectHtml({ id: index, ...project });
+
+  document.getElementById(`project-button-${index}`).addEventListener('click', () => {
+    projectModal.style.display = 'flex';
+  });
+});
+
+projectModalCloseButton.addEventListener('click', () => {
+  projectModal.style.display = 'none';
+});
```
Comment (CollinsTatang) [inline src/js/projects.js:87]: Hi @teckim 
As mentioned by the previous code reviewer, users still find it difficult to close the project details pop-up page. Please kindly fix this issue if not the user will find it difficult to return to the home page after clicking on the `See Project` button

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @teckim , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/projects.js (hunk 2 above)
Comment (teckim) [reply src/js/projects.js:87]: Hi @CollinsTatang, Kindly check again, because it works just find on my end.

**Review by gemmen29 — APPROVED** [review]
Hi @teckim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/projects.js (hunk 2 above)
Comment (CollinsTatang) [reply src/js/projects.js:87]: Ok @teckim

**Pull request merged**